appveyor.yml

```diff
@@ -1,10 +1,6 @@
 ﻿image:
-    - Visual Studio 2017
     - Visual Studio 2019
-before_build:
-    - nuget restore
-
-before_test:
-    - nuget install xunit.runner.console -version 2.4.1
+build_script:
+    - dotnet build -c Release
 test_script:
-    - xunit.runner.console.2.4.1\tools\net46\xunit.console.exe src\WpfMath.Tests\bin\Debug\net461\WpfMath.Tests.dll
+    - dotnet test -c Release
```

Comment:
For `dotnet test` to work, you probably need some additional references.

Reply:
It works, however, some test failed due to float-point precision issues.

Reply:
And, yes, Release build creates troubles for ApprovalTests, that's why we were testing in Debug.

Reply:
Fixed via https://github.com/ForNeVeR/wpf-math/pull/219/commits/ace9c359d42e62a513ffd0ca1415d8084d29022d